cmd/podman/common/create.go

```diff
@@ -277,7 +277,7 @@ func DefineCreateFlags(cmd *cobra.Command, cf *ContainerCLIOpts) {
 	createFlags.StringSliceVar(
 		&cf.GroupAdd,
 		groupAddFlagName, []string{},
-		"Add additional groups to join",
+		"Add additional groups to the primary user process. 'keep-groups' allows container to use suplementary groups.",
```

Comment:
Primary user process? Should this be "container process"?

Reply:
Fixed